src/overload_hessian.jl

```diff
@@ -1,85 +1,193 @@
 ## 1-to-1
-for fn in ops_1_to_1
-    @eval function Base.$fn(t::T) where {T<:GlobalHessianTracer}
-        if is_seconder_zero_global($fn)
-            if is_firstder_zero_global($fn)
-                return empty(T)
-            else
-                return t
-            end
+function hessian_tracer_1_to_1(
+    t::T, is_firstder_zero::Bool, is_seconder_zero::Bool
+) where {T<:HessianTracer}
+    if is_seconder_zero
+        if is_firstder_zero
+            return empty(T)
         else
-            return T(t.grad, t.hess ∪ (t.grad × t.grad))
+            return t
         end
+    else
+        return T(gradient(t), hessian(t) ∪ (gradient(t) × gradient(t)))
```

Comment:
and here

Reply:
Either answered [here](https://github.com/adrhill/SparseConnectivityTracer.jl/pull/65#discussion_r1604009803) or I don't understand your point.